modules/core/arrow-test/src/main/kotlin/arrow/test/laws/TraverseLaws.kt

```diff
@@ -47,17 +47,6 @@ class TIF {
 }
 
 object TraverseLaws {
-  // FIXME(paco): this implementation will crash the inliner. Wait for fix: https://youtrack.jetbrains.com/issue/KT-18660
```

Comment:
@pakoito hope is ok to remove this, or shall we refactor it somehow?

Reply:
The original bug is fixed, maybe it's worth bringing it back.